docs/org/language-reference/compile-blocks.org

```diff
@@ -3,7 +3,7 @@
 The compile keyword has two arguments:
 
 - A name of an expression to be compiled.
-- A set of compilation rules using the format (=backend= → =string=)
+- A set of compilation rules using the format (=backend= ↦ =string=)
```

Comment:
Better if we use "->" instead of the Unicode symbol here.

Reply:
The format uses `↦`, do we have a non unicode equivalent for \mapsto?

Reply:
😂 thanks - I completely forgot about this!